src/GafferImageUI/ImageView.cpp

```diff
@@ -1601,10 +1620,14 @@ void ImageView::insertConverter( Gaffer::NodePtr converter )
 
 	PlugPtr newInput = converterInput->createCounterpart( "in", Plug::In );
 	setChild( "in", newInput );
-
+		
 	NodePtr preprocessor = getPreprocessor();
 	Plug::OutputContainer outputsToRestore = preprocessor->getChild<ImagePlug>( "in" )->outputs();
 
+	// TODO - this is now actually broken, because any converters should be downstream of the
+	// comparison switch, so they affect both sides of the wipe.  Hopefully John has an idea on how
+	// to approach this.
```

Comment:
Oh, OK.  Maybe it's fine for how this actually gets used.  I was picturing a converter with an image input, where it would be weird to have it only applied to one side of the wipe, but maybe there's no use case for this ... we certainly have better ways to apply color transforms.